### PR TITLE
FE-417: adjust margin for button overlap

### PR DESCRIPTION
**Pull request by palashkaria**
### Scope & Purpose

- [x] :hankey: Bugfix

### Checklist

- [x] Tests
  - [x] Manually tested

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-417

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/notificationView.js [hunk 1]
```diff
@@ -15,7 +15,10 @@
       this.collection.bind('reset', this.renderNotifications.bind(this));
 
       window.setTimeout(function () {
-        if (frontendConfig.authenticationEnabled === false && frontendConfig.isCluster === false && arangoHelper.showAuthDialog() === true) {
+        if (frontendConfig.authenticationEnabled === false &&
```
Comment (nikhil-varma) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/notificationView.js:18]: Has this changed? Or just pushed to another line?

**Review by nikhil-varma — APPROVED** [review]
Minor question. Looks good otherwise.

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/notificationView.js (hunk 1 above)
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/notificationView.js:18]: Just pushed to another line, for better readability

Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/notificationView.js:18]: I can revert this

**Pull request merged**